README.md

```diff
@@ -1,171 +1,577 @@
+```csharp
+new BookingSeed().WithNew(b => b.Coupon, new CouponSeed().With(...));
+```
+
+#### Seed a grandparent entity
+```csharp
+new BookingSeed().WithNew(b => b.Facility.Building);
+```
+
+> [!NOTE]
+> This isn't necessary if these relationships are mandatory.
+
+If any parent in the property chain is optional, you will get a null reference exception. You can work around this as follows:
+```csharp
+new BookingSeed()
+    .WithNew(b => b.Parent.OptionalParent)// This is necessary to ensure `OptionalParent` is not null before seeding further properties on it.
+    .WithNew(b => b.Parent.OptionalParent.Parent);
 ```
 
-### Configuration
+### `WithNew` (multiple entities)
 
-There are two ways to configure seeding behaviour. All `DbSeed` classes have an overridable `Count` property which can be used to set the number of entities a given seed fixture should produce.
+#### Set a navigation property to the provided seeds in order
+```csharp
+List<MemberSeed> memberSeeds = [ new MemberSeed(), new MemberSeed() ];
+
+new BookingSeed().WithNew(
+    b => b.Coupon,
+    // First 3 bookings are for the first member
+    memberSeeds[0], 
+    memberSeeds[0], 
+    memberSeeds[0], 
+    // Remaining 2 bookings are for the second member
+    memberSeeds[1], 
+    memberSeeds[1]);
+```
 
-Additionally, the more recommended approach would be to use the application settings in web.config or equivalent. Settings can be specified either with or without the namespace:
+> [!NOTE]
+> If all you need is for the parents to be different, you can use [WithDifferent](#withdifferent-multiple-entities) instead.
 
-```xml
-<configuration>
-    <appSettings>
-        <add key="seed:Holiday" value="10" />
-        <add key="seed:Job" value="10" />
-        <add key="seed:Person" value="100" />
-    </appSettings>
-</configuration>
+#### Set a navigation property to the same value for all entities:
+```csharp
+new BookingSeed().WithNew(
+    // All bookings get the same member with the provided first name.
+    b => b.Member, 
+    new MemberSeed().With(m => m.FirstName, "Member Name"));
 ```
 
-## Audacia.Seed.AutoFixture
+### `WithDifferent` (multiple entities)
+You can only use `WithDifferent` if you are seeding more than one entity. As such, the library will throw an exception if passed into `Seed` rather than `SeedMany`.
+#### Specify that two entities should not share a parent:
+```csharp
+new BookingSeed().WithDifferent(b => b.Member);
+```
+#### Specify that two entities should not share a grandparent (and therefore parent):
+```csharp
+new BookingSeed().WithDifferent(b => b.Member.MembershipGroup);
+```
 
-This library allows for seed fixtures to be used with the popular AutoFixture library.
-A single seed can be configured as follows:
+### `WithExisting`
+Use `WithExisting` to seed an entity that already exists in the database. Use this when you've already seeded other entities, or you're testing against a readily-seeded database.
```

Comment:
I'm not sure what the phrase "readily-seeded" means. Do you mean a database which has already been seeded?